app/src/main/java/com/tiptopgoodstudio/androidresources/MainActivity.java

```diff
@@ -1,26 +1,111 @@
 package com.tiptopgoodstudio.androidresources;
 
 import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
 import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.MotionEvent;
 import android.view.View;
-import android.widget.Button;
+import android.view.ViewGroup;
 
 public class MainActivity extends AppCompatActivity {
 
+    private BottomNavigationView mBottomNavigationView;
+
+    private SectionAdapter mSectionAdapter;
+
+    private ViewPager mViewPager;
+
+    MenuItem prevMenuItem;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        mSectionAdapter = new SectionAdapter(getSupportFragmentManager());
+
+        // Set up the ViewPager with the section adapter.
+        mViewPager = (ViewPager) findViewById(R.id.viewpager);
+        setupViewPager(mViewPager);
+
+        mBottomNavigationView = (BottomNavigationView) findViewById(R.id.bottom_navigation);
+
+        mBottomNavigationView.setOnNavigationItemSelectedListener(
+                new BottomNavigationView.OnNavigationItemSelectedListener() {
+                    @Override
+                    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                        switch (item.getItemId()) {
+                            case R.id.home:
+                                mViewPager.setCurrentItem(0);
+
+                                break;
+                            case R.id.resources:
+                                mViewPager.setCurrentItem(1);
+
+                                break;
+                            case R.id.settings:
+                                mViewPager.setCurrentItem(2);
+
+                                break;
+                        }
+                        return false;
+                    }
+                });
+
+        mViewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+            }
+
+            @Override
+            public void onPageSelected(int position) {
+                if (prevMenuItem != null) {
+                    prevMenuItem.setChecked(false);
+                }
+                else
```

Comment:
else on the same line as } 

`} else {`

Reply:
Got it, made change in the coming commit.